.golangci-required.yml

```diff
@@ -10,6 +10,13 @@ linters-settings:
       - errors
       # use "github.com/golang/protobuf/proto" instead
       - github.com/gogo/protobuf/proto
+      # Use service/k8sclient/internal/... instead
+      - k8s.io/api/core/v1
+      - k8s.io/apimachinery/pkg/api/resource
+      - k8s.io/apimachinery/pkg/apis/meta/v1
+      - k8s.io/apimachinery/pkg/apis/runtime
+      - k8s.io/apimachinery/pkg/apis/types
+      - k8s.io/apimachinery/pkg/apis/util/intstr
```

Comment:
Those changes look like a result of bad merge – they should not be in that PR

Reply:
Fixed merge conflicts and they lost.